src/App.js

```diff
@@ -44,32 +47,32 @@ function App() {
             <a
               className="App-link"
               href="https://reactjs.org/"
-              rel="noopener noreferrer"
-              target="_blank">
+              target="_blank"
+              rel="noopener noreferrer">
               React
             </a>
             <span>, </span>
             <a
               className="App-link"
               href="https://redux.js.org/"
-              rel="noopener noreferrer"
-              target="_blank">
+              target="_blank"
+              rel="noopener noreferrer">
               Redux
             </a>
             <span>, </span>
             <a
               className="App-link"
               href="https://redux-toolkit.js.org/"
-              rel="noopener noreferrer"
-              target="_blank">
+              target="_blank"
+              rel="noopener noreferrer">
               Redux Toolkit
             </a>
             ,<span> and </span>
             <a
               className="App-link"
               href="https://react-redux.js.org/"
-              rel="noopener noreferrer"
-              target="_blank">
+              target="_blank"
+              rel="noopener noreferrer">
               React Redux
             </a>
           </span>
```

Comment:
the changes are not scope of this user story. Please add only the testimonialForm route